W-12769621 Update policies-custom-flex-implement-rust.adoc (#377)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/policies-custom-flex-implement-rust.adoc

----
rustup target add wasm32-wasi
----
+

> All third-party libraries included in your policy must be compatible with the `wasm32-wasi` Rust compilation target

Reviewer: ```suggestion
All third-party libraries included in your policy must be compatible with the `wasm32-wasi` Rust compilation target.
```

+
[IMPORTANT]
====

> It is possible that some libraries compile properly to the wasm32-wasi target but don’t work properly when deployed to Flex Gateway. Example errors of this are:

Reviewer 2: ```suggestion
Some libraries might compile properly to the wasm32-wasi target but might not work properly when deployed to Flex Gateway. Example deployment errors when this happens are:
```

* `Failed to load Wasm module due to a missing import`
* `Wasm VM failed to initialize Wasm code`
* `Plugin configured to fail closed failed to load`

> In this case, contact the owners of the third-party library or use a different library.

Reviewer 2: ```suggestion
To resolve these issues, contact the owners of the third-party library or use a different library.
```
